locals.tf

```diff
@@ -0,0 +1,131 @@
+
+## Managed By : CloudDrove
+## Copyright @ CloudDrove. All Right Reserved.
+
+## Vritual Network and Subnet Creation
```

Comment:
Can we remove this part and above one also

Reply:
Removed